src/libraries/System.Reflection.Metadata/tests/System.Reflection.Metadata.Tests.csproj

```diff
@@ -24,6 +26,10 @@
              Link="Common\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs" />
     <Compile Include="$(CommonTestPath)System\IO\TempFile.cs"
              Link="Common\System\IO\TempFile.cs" />
+    <Compile Include="$(CommonPath)System\Security\Cryptography\IncrementalHash.netfx.cs"
```

Comment:
S.R.Metadata is using `IncrementalHash` in a single place, and it's for unit tests. If it's better or more acceptable, we can change `CalculateSha1` to just not use `IncrementalHash` like we did for Microsoft.Bcl.Cryptography.

Reply:
I tried avoiding the dependency but couldn't get it working as the test code incrementally calculates the hash across segments: https://github.com/dotnet/runtime/blob/3f2af4604147dd5299b9fefcdd6ac38012b72838/src/libraries/System.Reflection.Metadata/tests/TestUtilities/SigningUtilities.cs#L32-L46

Do you think we could use a different code path here?

Reply:
I don't know why this method is creating a `MemoryStream` and doing nothing with the stream... but.. we might as well use it.

```C#
public static byte[] CalculateSha1(IEnumerable<Blob> content)
{
    var stream = new MemoryStream();

    foreach (var blob in content)
    {
        var segment = blob.GetBytes();
        stream.Write(segment.Array, segment.Offset, segment.Count);
    }

    stream.Position = 0;

    using (SHA1 sha1 = SHA1.Create())
    {
        return sha1.ComputeHash(stream);
    }
}
```